commons-numbers-combinatorics/src/main/java/org/apache/commons/numbers/combinatorics/FactorialDouble.java

```diff
@@ -64,8 +64,8 @@ private FactorialDouble(int numValues,
         }
 
         // Copy available values.
-        for (int i = beginCopy; i < endCopy; i++) {
-            factorialsDouble[i] = cache[i];
+        if (endCopy - 2 >= 0) {
```

Comment:
There are now null pointer fails in the travis build. You cannot assume `endCopy - 2 >= 0`. Try changing this to `endCopy - beginCopy >= 0`.

Reply:
I think same result